### PR TITLE
feature(ui-ux): future swap from portfolio v2

### PR DESCRIPTION
**Pull request by JoshuaC817**
#### What kind of PR is this?:

/kind feature

#### What this PR does / why we need it:
Update future swap screens from portfolio to v2 (listing and withdraw)

#### Which issue(s) does this PR fixes?:

Fixes #

#### Developer Checklist:

- [x] Read your code changes at least once
- [x] Tested on iOS/Android device (e.g, No crashes, library supported etc.)
- [x] No console errors on web
- [x] Tested on Light mode and Dark mode*
- [x] Your UI implementation visually matched the rendered design*
- [x] Unit tests*
- [x] Added e2e tests*
- [x] Added translations*

### CODE REVIEW

**Review by pierregee — CHANGES_REQUESTED** [review]
Just some nitpicks and one bug that is existing in v1
image
Can disable the button if more than your future swap balance, it looks like it's a bug in v1. Let's fix it in v2 😱

File: mobile-app/app/screens/AppNavigator/screens/Portfolio/screens/WithdrawFutureSwapScreenV2.tsx [hunk 1]
```diff
@@ -0,0 +1,332 @@
...
+  const [fee, setFee] = useState<BigNumber>(new BigNumber(0.0001))
+
+  const toast = useToast()
+  const TOAST_DURATION = 2000
+  const [transactionCardStatus, setTransactionCardStatus] = useState<TransactionCardStatus>()
+  const [isInputFocus, setIsInputFocus] = useState<boolean>(false)
+  const [hasError, setHasError] = useState<boolean>(false)
+  const modalSortingSnapPoints = {
+    ios: ['50%'],
+    android: ['50%']
```
[the remark this answers is not in the document]
Comment (JoshuaC817) [reply mobile-app/app/screens/AppNavigator/screens/Portfolio/screens/WithdrawFutureSwapScreenV2.tsx]: adjusted to 45% for both ios and android

File: mobile-app/app/screens/AppNavigator/screens/Portfolio/screens/ViewSwapInfo.tsx [hunk 2]
```diff
@@ -0,0 +1,110 @@
+import { Platform, View } from 'react-native'
```
[the remark this answers is not in the document]
Comment (JoshuaC817) [reply mobile-app/app/screens/AppNavigator/screens/Portfolio/screens/ViewSwapInfo.tsx:1]: fixed the space below, couldn't reproduce the line on top, could you try again with the new changes?

**Comment by JoshuaC817** [comment]
> Just some nitpicks and one bug that is existing in v1 image Can disable the button if more than your future swap balance, it looks like it's a bug in v1. Let's fix it in v2 😱

issue fixed

**Review by pierregee — DISMISSED** [review]
LGTM! 🚀 
image

**Comment by thedoublejay** [comment]
Sorry need to update conflicts :D @JoshuaC817

**Pull request merged**